frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardGraph.tsx

```diff
@@ -42,7 +42,7 @@ export const MonitoringDashboardGraph: React.FC<MonitoringDashboardGraphProps> =
         <DashboardCardTitle>{title}</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
-        <PrometheusGraphLink query={query}>
+        <PrometheusGraphLink query={query} ariaChartLinkLabel={title}>
```

Comment:
You generally want something that describes the link itself rather than just the graph title. We've used `View {{title}} metrics in query browser` in other places.

https://github.com/openshift/console/blob/7e39ae0354a55d79518bfbd3f307439230dcbd46/frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx#L70